README.md

```diff
@@ -102,3 +102,13 @@ Zivid Samples are distributed under the [BSD license](https://github.com/zivid/c
 
 [ci-badge]: https://img.shields.io/azure-devops/build/zivid-devops/5e76c4a5-26ad-4cbb-8ab5-b9588e1ed2b2/4
 [ci-url]: https://dev.azure.com/zivid-devops/cpp-extra-samples/_build/latest?definitionId=4&branchName=master
+
+## Development
+
+To run continuous integration locally, use [Docker](https://www.docker.com). With Docker installed, run this command:
+```
+docker run -it -v <unixy-repo-path>:/host -w /host/continuous-integration/linux ubuntu:18.04 bash -c "./setup.sh && ./lint.sh && ./build.sh"
+```
+Where <unixy-repo-path> is the unixy path to the repo on your computer. On Linux, use `$PWD` for this. On Windows you need to translate the windowsy path to a unixy one (e.g. `/c/Users/alice/Documents/cpp-extra-samples`).
+
+Running this takes a long time, since `setup.sh` takes several minutes to complete. Often it's useful to run CI locally many times, as you keep fixing your code. In that case, remove everything after `bash` in the above command. Then type `setup.sh` once to set it up, after which you can quickly run `./lint.sh && ./build.sh` repeatedly to check your code.
```

Comment:
Maybe this shoult be the only recommended way. Then this Readme section can be simplified.

Reply:
Good point! Fixed.